kodilla-testing/src/main/java/com/kodilla/testing/TestingMain.java

```diff
@@ -1,9 +1,28 @@
 package com.kodilla.testing;
 
-import java.sql.SQLOutput;
+import com.kodilla.testing.calculator.Calculator;
+import com.kodilla.testing.user.SimpleUser;
 
 public class TestingMain {
     public static void main(String[] args) {
         System.out.println("Moduł 6. Wprowadzenie to testowania oprogramowania");
+
+        SimpleUser simpleUser = new SimpleUser("theForumUser");
+
+        String result = simpleUser.getUsername();
+
+        if (result.equals("theForumUser")) {
+            System.out.println("test OK");
+        } else {
+            System.out.println("Error!");
+        }
+
+        System.out.println("Test - pierwszy test jednostkowy:");
+
+        Calculator calculator = new Calculator();
+
+        System.out.println(calculator.add(3, 5));
```

Comment:
To nie jest testowanie.

Reply:
Ok, poprawię. Zauważyłem już wcześniej, że źle do tego podszedłem, ale nie chciałem dawać następnego commita, bo nie wiedziałem, czy będziesz widział poprzednie zmiany.